src/components/pages/Offers/Offer/Stocks/ActivationCodesUploadDialog/ActivationCodesCsvForm/ActivationCodesCsvForm.jsx

```diff
@@ -41,6 +43,31 @@ export const ActivationCodeCsvForm = forwardRef(function ActivationCodeCsvForm(
         </div>
       </div>
       <div className="activation-codes-upload-separator" />
+      <div className="activation-codes-upload-template-section">
+        <p className="activation-codes-upload-gabarit">
+          {'Gabarits'}
+        </p>
+        <p>
+          <a
+            className="quaternary-link"
+            href={
+              process.env.PUBLIC_URL +
+              '/csvtemplates/CodesActivations-[NomOffre]-[NomDeVotreStructure].csv'
```

Comment:
```suggestion
              '/csvtemplates/CodesActivations-[NomOffre]-[NomDeVotreStructure].csv'
```
Est-ce qu'on voulait que [NomOffre] et [NomDeVotreStructure] soit remplacé ?

Reply:
renommé en **CodesActivations-Gabarit.csv**